notification-portlet-webcomponents/notification-component-icon/src/NotificationIcon.js

```diff
@@ -0,0 +1,122 @@
+import React, {Component} from 'react';
+import {
+  Dropdown,
+  DropdownMenu,
+  DropdownToggle,
+  DropdownItem,
+} from 'reactstrap';
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+import {translate} from 'react-i18next';
+
+class NotificationIcon extends Component {
+  constructor(props) {
```

Comment:
constructor here is not required if you're going the method = () => route

Reply:
Will do

Reply:
updated